modules/cert-manager-about.adoc

```diff
@@ -17,5 +17,7 @@ The {cert-manager-operator} provides the following features:
 
 [IMPORTANT]
 ====
-Do not attempt to use more than one cert-manager Operator in your cluster. If you have a community cert-manager Operator installed in your cluster, you must uninstall it before installing the {cert-manager-operator}.
+Do not attempt to use both cert-manager Operator for {product-title} and community {cert-manager-operator} at the same time in your cluster.
```

Comment:
```suggestion
Do not attempt to use both {cert-manager-operator} for {product-title} and community cert-manager operator at the same time in your cluster.
```

Reply:
I think it should be fine to not remove the second sentence: `If you have a community cert-manager Operator installed in your cluster, you must uninstall it before installing the {cert-manager-operator}.`

Reply:
ack.